docs/modules/ROOT/nav.adoc

```diff
@@ -1,15 +0,0 @@
-// TODO: Edit navigation
-* xref:index.adoc[Introduction]
-* https://github.com/vshn/go-bootstrap/releases[Changelog,window=_blank] 🔗
-
-.Tutorials
-//* xref:tutorials/example.adoc[Example Tutorial]
-
-.How To
```

Comment:
This completely removes the nav, please keep it and add the link to the parameter references and the changelog

Reply:
I've re-added it but antora just quietly ignores it :shrug: 

It is not relevant however because that documentation gets integrated into the component hub which has its own navigation.

Reply:
> but antora just quietly ignores it
strange, it's not applied for me as well, but in `component-appcat` it does... I suspect some caching issues.

But I don't understand what you mean by not relevant for Hub. The Hub certainly uses the navigation within a component, see example here (lots of articles in the menu left): https://hub.syn.tools/keycloak/index.html

Reply:
Counter point: https://hub.syn.tools/cloudscale-metrics-collector/index.html works just fine even though there is no nav.adoc present in https://github.com/vshn/cloudscale-metrics-collector/tree/master/docs/modules/ROOT either.

I'd just merge it to master, wait for hub.syn.tools to update itself and fiddle with this some more if the result isn't what we want.